Userland/Libraries/LibGUI/TableView.cpp

```diff
@@ -174,7 +174,8 @@ void TableView::keydown_event(KeyEvent& event)
     auto is_delete = event.key() == Key_Delete;
     auto is_backspace = event.key() == Key_Backspace;
     auto is_clear = is_delete || is_backspace;
-    if (is_editable() && edit_triggers() & EditTrigger::AnyKeyPressed && (event.code_point() != 0 || is_clear)) {
+    auto is_clipboard_action = (event.modifiers() & KeyModifier::Mod_Ctrl) && (event.key() == Key_Z || event.key() == Key_X || event.key() == Key_C || event.key() == Key_Y);
```

Comment:
Assuming that those are actually bound to clipboard actions doesn't sound right.
However, I'd argue that Ctrl+anything should never insert text into a GUI field anyway, so perhaps change it to that?